.gitignore

```diff
@@ -1,27 +1,29 @@
-# Docker project generated files to ignore
-#  if you want to ignore files created by your editor/tools,
-#  please consider a global .gitignore https://help.github.com/articles/ignoring-files
+# If you want to ignore files created by your editor/tools, please consider a
+# [global .gitignore](https://help.github.com/articles/ignoring-files).
+
 *.exe
 *.exe~
 *.gz
 *.orig
-test.main
 .*.swp
 .DS_Store
-# a .bashrc may be added to customize the build environment
+
+# local repository customization
+.direnvrc
 .bashrc
 .editorconfig
-.gopath/
-.go-pkg-cache/
+
+# top-level go.mod is not meant to be checked in
+/go.mod
+# build artifacts
 bundles/
-cli/winresources/**/winres.json
 cli/winresources/**/*.syso
+cli/winresources/**/winres.json
 cmd/dockerd/dockerd
```

Comment:
This is not the standard location the binary is built (using the makefile), so I guess this was added for cases where someone would manually build it.

If so, perhaps we need to add `cmd/docker-proxy/docker-proxy`

Hm... although we wouldn't know where the binary would land (running `go build ./cmd/docker-proxy/` would put the binary at the root. So perhaps be "less" specific for this one as well, and use `dockerd`, `docker-proxy` ?


Do we need to account for the `libnetwork/cmd/*` directories? Looks like libnetwork itself didn't (at least not for all of them), but did have some other ignores (possibly related to tests?); https://github.com/moby/libnetwork/blob/master/.gitignore

Reply:
I think we should just not account for either of them, I only left it as I was a bit unsure of any hidden implications.